helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/ClusterAccessor.java

```diff
@@ -282,6 +282,13 @@ public Response updateCluster(@PathParam("clusterId") String clusterId,
           return badRequest(e.getMessage());
         }
         break;
+      case purgeOfflineParticipants:
+        if (timeout == null || timeout < 0) {
+          helixAdmin.purgeOfflineInstances(clusterId, -1);
```

Comment:
-1 is a magic number and we should generally avoid having magic numbers..

it would look nicer if you could make the not set constant a public constant and use it here - that way it's easy to track what's going on and code is much more readable.